detection-rules/body_bec_covid_international_org_scam.yml

```
@@ -30,16 +31,34 @@ source: |
                       'interpol',
                       'treasury',
                       '\bFEMA\b',
+                      '\bIMF\b'
   )
   
   // and mention of covid in subject or body
-  and regex.icontains(body.current_thread.text, 'covid(.0,5}19)?\b')
+  and (
+    regex.icontains(subject.subject, 'covid(.{0,5}19)?\b')
+    or regex.icontains(body.current_thread.text, 'covid(.{0,5}19)?\b')
+  )
   
-  // urgent financial requests
-  and 2 of (
-    any(ml.nlu_classifier(body.current_thread.text).entities, .name == "urgency"),
-    any(ml.nlu_classifier(body.current_thread.text).entities, .name == "request"),
-    any(ml.nlu_classifier(body.current_thread.text).entities, .name == "financial")
+  // Check for compensation or award related language
+  and (
+    2 of (
+      any(ml.nlu_classifier(body.current_thread.text).entities, .name == "urgency"),
+      any(ml.nlu_classifier(body.current_thread.text).entities, .name == "request"),
+      any(ml.nlu_classifier(body.current_thread.text).entities, .name == "financial")
+    )
+    or regex.icontains(subject.subject, 
+                      'compensation.{0,20}(award|fund)',
+                      'covid.{0,20}(compensation|award)',
+                      'selected.{0,30}(compensation|award)',
+                      'claim your award',
+                      'reference no')
+    or regex.icontains(body.current_thread.text, 
+                      'compensation.{0,20}(award|fund)',
+                      'covid.{0,20}(compensation|award)',
+                      'selected.{0,30}(compensation|award)',
+                      'claim your award',
+                      'reference no')
```

`reference no` seems out of place
```suggestion
                      )
```

---

It was in the sample linked, but happy to rip it out

---

maybe something like this to tighten it up a bit more? 
```suggestion
                      'reference no\W\s*[^\s]*cov(?:id)?(?:.{0,5}19)?\b')
```